lib/Guideline/styles.scss

```diff
@@ -44,6 +44,7 @@ $guideline-title-size: $typo-size-large !default;
 
 .guideline.is-active {
   .guideline__body {
-    max-height: 400px;
+    max-height: 100%;
```

Comment:
I thought this would have handled the animation without the need for JS but I gather it doesn't 😂